### PR TITLE
Automatically detect language and add to the query

### PR DESCRIPTION
**Pull request by unique1o1**

### CODE REVIEW

File: subflow.py [hunk 1]
```diff
@@ -10,20 +10,35 @@ def run(self, edit):
...
-                if sublime.platform() == 'windows':
-                    output = output.replace('\r', '')
-
-                self.view.replace(edit, line, output)
+        if (len(cont) < 2):
+            pass
+        else:
+            lang = "in " + self.get_syntax()
+            cont = cont + lang
+            print(cont)
```
Comment (amitness) [inline subflow.py]: - Remove the print statement

**Review by amitness — CHANGES_REQUESTED** [review]

File: subflow.py (hunk 1 above)
Comment (unique1o1) [reply subflow.py]: ok done.

**Comment by amitness** [comment]
Thanks @unique1o1

**Comment by amitness** [comment]
Fixes #1

**Pull request merged**